batch/test/test_accounts.py

```diff
@@ -75,12 +75,12 @@ async def test_bad_token():
     token = session_id_encode_to_str(secrets.token_bytes(32))
     bc = await BatchClient.create('test', _token=token)
     try:
-        b = bc.create_batch()
-        j = b.create_job(DOCKER_ROOT_IMAGE, ['false'])
-        await b.submit()
+        builder = bc.create_batch()
+        builder.create_job(DOCKER_ROOT_IMAGE, ['false'])
+        b = await builder.submit()
         assert False, str(await b.debug_info())
     except aiohttp.ClientResponseError as e:
-        assert e.status == 401, str((e, await b.debug_info()))
+        assert e.status == 401
```

Comment:
Why remove the debugging information?

Reply:
b (as a Batch, not a BatchBuilder) won't actually exist if there's an error raised in this block, so the debug information will always be a None dereference.

Reply:
We should perhaps add debug_info to a BatchBuilder.